apps/modernization-ui/src/apps/patient/profile/labReport/LabReportTable.tsx

```diff
@@ -56,7 +56,7 @@ const asTableBody =
             },
             {
                 id: 3,
-                title: format(report?.collectedOn, 'MM/dd/yyyy') || null
+                title: internalizeDate(report?.collectedOn) || null
```

Comment:
`internalizeDate(report?.collectedOn)` You can use this directly as the `internalizeDate` function is returning the null value.